index.html

```diff
@@ -10,10 +10,27 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<!-- Navigation -->
 <body>
+        <nav class="navigation">
+            <div>
+                <img src="C:\Users\tate\Desktop\Lambda School\Projects\Sprint\Sprint-Challenge--User-Interface\img\lambda-black.png">
```

Comment:
Inserting the images this way will cause them not to show up in a live browser.  They'll only show up locally.

You only need the **directory where the images are + the file name**.  

In your case,for this project:
`\img\jumbo.jpg`